tools/perl-version-convert

```diff
@@ -70,8 +70,11 @@ function convert_version
 	# be converted to 0.XX.YY, but it was already integrated as 0.XXYY, so
 	# we need to follow it for now.
 	#
+	# convert "Mail-AuthenticationResults" from v.YYYYMMDD to v.YYYY.M.DD
+	#
 	[[ "$DIST" == "Authen-PAM" && "${VERSION:0:2}" == "0." ]] && ((${#VERSION} == 4)) && VERSION=$((1${VERSION:2:2} - 100))
 	[[ "$DIST" == "Crypt-PBKDF2" && "${VERSION:0:2}" == "0." ]] && ((${#VERSION} == 8)) && VER[1]=$((1${VERSION:2:6} - 1000000)) && VERSION=${VERSION:0:1}
+	[[ "$DIST" == "Mail-AuthenticationResults" && "${VERSION:0:2}" == "2." ]] && ((${#VERSION} == 10)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VERSION=${VERSION:0:1}
```

Comment:
Please move this down, so DISTs are sorted.

Reply:
should be solved